### ISSUE TITLE
New `New note...` should be gone if user switches to other note

### ISSUE THREAD

**Issue by pachlava** [Automattic/Simplenote-United#116]
### What
According to p1615308016122100-slack-C01KRRK0MLJ, it's expected that new notes with no changes should not be saved (as happens now in Electron / macOS / Android), and should not be moved to Trash (as happens on iOS). Instead, they should just disappear.

### Why
Consistency and saving screen space

**Comment by Erudition** [comment]
Agreed, I wondered why I kept having untitled empty notes, it was just from visiting the editor screen.
